CMakeLists.txt

```diff
@@ -77,6 +77,12 @@ if(NOT ONNX_NAMESPACE)
   set(ONNX_NAMESPACE "onnx")
 endif()
 
+# find_package Python has replaced PythonInterp and PythonLibs since cmake 3.12
+# Use the following command in the future; now this is only compatible with the latest pybind11
+# find_package(Python ${PY_VERSION} COMPONENTS Interpreter DevelopmentREQUIRED)
+find_package(PythonInterp ${PY_VERSION} REQUIRED)
+find_package(PythonLibs ${PY_VERSION} REQUIRED)
```

Comment:
Originally ONNX only uses `find_package(PythonInterp)` and `find_package(PythonLibs)` to find python library on Windows... Move here can help finding the right Python on Linux, but I am not sure whether this will break something in some platforms.

Reply:
Should they be required only when BUILD_ONNX_PYTHON  is ON?

Reply:
Make sense. Added a condition for this. Thanks.